01-Git_Bash_Make/makefile

```diff
@@ -0,0 +1,15 @@
+COMPILER=gcc
+OBJS=main.o random_func.o
+
+install: ${OBJS}
```

Comment:
install is standard target, which is used for installing not for compiling. Please, rename to "game" or etc